spec/features/restaurants/visitor_tries_update_restaurant_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+feature 'prevent unuathorized updates to restaurant' do
+  context 'as a visitor' do
+    scenario 'I cannot save a restaurant' do
+      visit new_restaurant_path
+      expect(page).to not_have_link("Create Restaurant")
+    end
+
+    scenario 'I cannot edit a restaurant' do
+      visit edit_restaurant_path("1")
+      expect(page).to not_have_link("Edit")
+    end
+
+    scenario 'I cannot delete a restaurant' do
+      visit destroy_restaurant_path("1")
+      expect(page).to have_link("Delete")
+    end
+  end
+
+  context 'As a signed in user viewing a restaurant I did not created' do
+    before :each do
+      restaurant = FactoryGirl.create(:restaurant)
```

Comment:
In this case, I'd like to assign the restaurant variable using the before(:each) block and use it within the scenarios. Is there a change in scope between the two? ie, do I need to make restaurant an instance variable or is this just Hound being overzealous?